client/src/components/coffee/AddCoffeeForm.tsx

```diff
@@ -0,0 +1,136 @@
+import React, { FormEvent, useState } from 'react';
+import styled from 'styled-components';
+
+import { CoffeeApi } from '../../api/CoffeeApi';
+import { ICoffeeProfile } from '../../contracts';
+import { WindowUtility } from '../../util';
+
+const StyledForm = styled.form`
+  display: flex;
+  flex-direction: column;
+  max-width: 24rem;
+  align-items: center;
+  border: 1px black solid;
+  border-radius: 8px;
+  padding: 12px;
+`;
+
+const StyledLabel = styled.label`
+  margin: 0.5rem;
+`;
+
+const StyledInput = styled.input`
+  margin-left: 1rem;
+  border-style: ridge;
+`;
+
+const StyledTextArea = styled.textarea`
+  margin-left: 1rem;
+  border-style: ridge;
+`;
+
+const StyledInputButton = styled.input`
+  border-radius: 4px;
+  border-style: outset;
+`;
+
+type AddCoffeeFormProps = {
+  userId: string;
+};
+
+const AddCoffeeForm = ({ userId }: AddCoffeeFormProps) => {
```

Comment:
Is there no validation on the input?

Reply:
Not yet, I just wanted to get it working first, another ticket coming. #28

Reply:
Makes sense.